### DISCUSSION TITLE
Duel Decks: Pirates vs. Ninjas (PVN)

### DISCUSSION THREAD

**Discussion by yandere-sliver** [andrewgioia/keyrune#258] [General]
![PVN_Ramirez_DePietro](https://user-images.githubusercontent.com/25598912/97764759-fabbc080-1b0f-11eb-9564-ff5fd1017a41.jpg)

I just realized that this April fools joke actually had a set symbol - a shuriken and a hook

**Comment by andrewgioia** [comment]
@yandere-sliver is this a real card from WOTC or a fan-created card?

**Comment by yandere-sliver** [comment]
Hi Andrew 
it is from WotC but it was never printed to my knowledge. It was from an April Fools Joke they pulled one year and created a few alternate artwork cards, a set symbol as well as decklists.

**Comment by scarletcs** [comment]
Not a real released product. Posted on the mothership here:

[A Special Product Announcement — April 1, 2008](https://magic.wizards.com/en/articles/archive/special-product-announcement-2008-04-01)

**Comment by andrewgioia** [comment]
Wayback grabbed this page, which seems to be gone: https://web.archive.org/web/20220529034319/https://magic.wizards.com/en/articles/archive/special-product-announcement-2008-04-01

Some additional cards for reference, though the symbol is still really low res:

![1559_Higure](https://github.com/andrewgioia/keyrune/assets/221473/b88902db-b952-40b2-854a-5980350baf9d)
![1559_MistformUltimusPirate](https://github.com/andrewgioia/keyrune/assets/221473/05479f03-1811-4d6b-b2fc-ae818e47d0a7)

**Comment by RivalRowan** [comment]
Can't let [this artifact](https://www.youtube.com/watch?v=UPt4XF5y0HY) be lost to time.
It adds no value to finding a high res version of the PVN symbol though.

**Comment by andrewgioia** [comment]
Wow! What a great time in the game that was where WOTC made a video like that. I wish they still had that sense of humor, everything is so corporate now!

**Comment by andrewgioia** [comment]
Moving this (and all older v4 issues) to discussions!
